L1Trigger/L1TCalorimeter/python/caloStage2Params_2017_v1_8_2_updateHFSF_v6MET_cfi.py

```diff
@@ -0,0 +1,228 @@
+#
```

Comment:
hi @rekovic - all of these caloStage2Params* files are really still relevant? (we do have a code versioning system after all..)